### DISCUSSION TITLE
[Feature Request] Group Chats and Public Threads

### DISCUSSION THREAD

**Discussion by MuckT** [WalletConnect/walletconnect-monorepo#2428] [Developer Support] [answered]
### Description

- As as user I'd like to join create threads that were open to the public e.g. #WalletConnectChatSupport or #EthNftMintTracker
- As as user I'd like to be able to create or be invited to a group chat where only invited members could join.

**Comment by Elyniss** [comment, marked as answer]
Hey! Thanks for raising a feature request! 

Both of these use cases seem to be group chats, with the only distinction of one being a private group chat and the other being public. We do not plan to have group chats as part of the Chat API feature set. We quickly realised that doing group chats in an encrypted, decentralised and reliable manner is complex and very engineering capacity-consuming.
